src/lib/divider/divider.md

```diff
@@ -0,0 +1,55 @@
+`<mat-divider>` is a component that allows for Material styling of a line separator with various orientation options.
+
+<!-- example(divider-overview) -->
+
+
+### Simple divider
+
+An `<mat-divider>` element can be used on its own to create a horizontal or vertical line styled with a Material theme
+
+```html
+<mat-divider></mat-divider>
+```
+
+### Inset divider
+
+Add the `inset` attribute in order to set whether or not the divider is an inset divider.
+
+```html
+<mat-divider [inset]="true"></mat-divider>
+```
+
+### Vertical divider
+
+Add the `vertical` attribute in order to set whether or not the divider is vertically-oriented.
+
+```html
+<mat-divider [vertical]="true"></mat-divider>
+```
+
+
+### Lists with inset dividers
+
+Dividers can be added to lists as a means of separating content into distinct sections.
+Inset dividers can also be added to provide the appearance of distinct elements in a list without cluttering content
+like avatar images or icons. If combining both, please make sure to avoid adding an inset divider to the last element
+in a list, because it will overlap with the section divider.
```

Comment:
What about:

> <strike>If combining both, please m</strike>Make sure to avoid adding an inset divider to the last element in a list, because it will overlap with the section divider

Reply:
Fixed